src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/_params.py

```diff
@@ -58,16 +58,32 @@ def list_app_service_plan_names(prefix, action, parsed_args, **kwargs):#pylint:
 register_cli_argument('webapp deployment slot', 'auto_swap_slot', help='target slot to swap', default='production')
 register_cli_argument('webapp deployment slot', 'disable', help='disable auto swap', action='store_true')
 
-switch_options = ['on', 'off']
-register_cli_argument('webapp log set', 'application_logging', choices=switch_options, type=str.lower, help='configure application logging to file system')
+two_states_switch = [True, False]
+def two_states_switch_type(value=None):
+    return value.lower() == 'true' if value else None
```

Comment:
`None` is expected, but the logic here should be improved to reject invalid values. I have let @Johanste review the new logic and committted